packages/client/src/entity-actions/edit-column-action.tsx

```diff
@@ -130,9 +134,13 @@ export function registerActionsForUpdatableColumns(columns: ColumnInfo[], entity
         const v = value ?? data?.[field];
         patchPayload[field] = expressions[`${v}`]?.() ?? v;
       });
-
-      // entityApi.patch(row.id, patchPayload)
-      console.log({ patchPayload, row });
+      const orgId = UserSelector.orgId(store.getState())!;
+      const endpoint = entityApi[entityType].endpoints.patch;
+      await store.dispatch(endpoint.initiate({
+        orgId,
+        rowId: row.id,
+        payload: patchPayload,
+      }));
```

Comment:
I think it may be better to pass in the `patch` function after setting up the hook in the caller's component. That way the caller can easily listen for errors if necessary, and the behavior might be a little more transparent. It's probably also not a best practice to access the store directly like this outside of an action.

So it might look something like this instead of passing in the entity type...

```typescript
  const [patch, {
    error: patchError,
  }] = entityApi[entityType].usePatchMutation();

  useEffect(() => {
    registerActionsForUpdatableColumns(allowedColumns, patch);
  }, [allowedColumns, patch]);
```

Reply:
This actually breaks a few of the hooks rules, which is why we took the above approach (see code in context).

Reply:
Updating this to use hooks requires refactoring that may be outside the scope of this task.